cpp/include/Ice/Initialize.h

```diff
@@ -117,7 +117,7 @@ ICE_API PropertiesPtr createProperties();
  * @return A new property set initialized with the property settings
  * that were removed from the argument vector.
  */
-ICE_API PropertiesPtr createProperties(StringSeq& seq, const PropertiesPtr& defaults = 0);
+ICE_API PropertiesPtr createProperties(StringSeq& seq, const PropertiesPtr& defaults = nullptr);
```

Comment:
I am wonder if we should just remove these functions that are now equivalent to a call to `make_shared<Properties>(...)`